test/functional/run/print-interface.glu

```diff
@@ -0,0 +1,36 @@
+//
+// RUN: gluc --print-interface %s | FileCheck -v %s
+//
+
+struct S {
+    a: Int,
+    public b: Int,
+}
+
+enum E {
+    A,
+    B,
+    C,
+}
+
+func foo(s: S, @unused e: E) -> Int {
+    let result = s.a + s.b;
+    return result;
+}
+
+@linkage_name("borabora")
+public func bar() -> Void {
+    // do nothing
+}
+
+// CHECK:private struct S {
+// CHECK-NEXT:    private a: Int32
+// CHECK-NEXT:    public b: Int32
+// CHECK-NEXT:}
+// CHECK-NEXT:private enum E {
+// CHECK-NEXT:    private A,
+// CHECK-NEXT:    private B,
+// CHECK-NEXT:    private C,
+// CHECK-NEXT:}
+// CHECK-NEXT:private func foo(s: S, @unused e: E) -> Int32;
+// CHECK-NEXT:@linkage_name("borabora") public func bar();
```

Comment:
you should put each of those check lines above the appropriate line in the source code, not group them at the end